src/Components/CardList/CardList.tsx

```diff
@@ -1,25 +1,25 @@
 //@ts-nocheck
-import React, {useState} from "react";
 import Card from "../Card/Card";
-
-const CardList = ({ data , title}) => {
-  console.log(data)
+interface CardListProps {
+  data: any,
+  title:string
+}
+const CardList = (props: CardListProps) => {
+  const {data, title} = props;
   return (
     <div className="commit__container">
-      <div className="title">Showing Results For <strong>{title} </strong></div>
-      {!data.length ? (
-        <div className="no-results">
-          Oh dear!<br></br> Your search returned no results.
+      {data.length ? (
+        <div className="title">
+          Showing Results For <strong>{title} </strong>
         </div>
-      ) : data.length === 1 ? (
+      ) : null}
+      {data.length === 1 ? (
         <Card title={data[0]} />
       ) : (
         <ul>
           {Object.entries(data).map((commit, i) =>
             data.length === i + 1 ? (
-              <div key={i}>
-                {commit.title}
-              </div>
+              <div key={i}>{commit.title}</div>
```

Comment:
Don't use the index this is what React uses by default. Use something off the commit.

Reply:
Im confused here. Why is data an object I feel like you need an array?